docs/njk/common.njk

```diff
@@ -10,8 +10,12 @@
 <br>
 {% endmacro %}
 
-{% macro embed(heading, src) %}
+{% macro embed(heading, src, omit_front_matter) %}
 <panel type="seamless" header="{{ icon_embedding }} {{ heading }}" minimized>
+{% if omit_front_matter %}
+<include src="{{ src }}" omitFrontmatter />
+{% else %}
 <include src="{{ src }}" />
+{% endif %}
```

Comment:
```suggestion
<include src="{{ src }}" {% if omit_front_matter %} omitFrontmatter {% endif %} />
```

What do you think of the above instead? This way we can omit the else block and reduce repetition.